tasks.py

```diff
@@ -66,7 +66,7 @@ def translations(c: Context):
 
 @task
 def schema(c: Context):
-    c.run("python manage.py spectacular --file api_schema.yaml")
+    c.run("python manage.py spectacular --file api-schema.yml --validate")
```

Comment:
I synced this to https://github.com/dimagi/open-chat-studio/blob/d3069eb96d5ade023f5555d72d677d41820f97ca/apps/api/tests/test_schema.py#L14

Reply:
Maybe we should update this comment to say use `inv schema`